### ISSUE TITLE
Mobile provisioning check fails for AppStore builds

### ISSUE THREAD

**Issue by Jaymassena** [padresmurfa/cryoprison#1]
Your code logic for iOS always expects there to be an embedded mobile provisioning file (except when simulator friendly is enabled). It turns out that AppStore apps do not have an embedded mobile provisioning file. In fact, checking for it and not finding it is one technique being used to determine if the current app is from the AppStore: https://stackoverflow.com/questions/16087779/detect-appstore-installation-of-ios-app

We had no false positives until we pushed out a release candidate via TestFlight and started getting mobile provisioning violations so we fortunately caught the issue before our customers did.

**Comment by mrstrong79** [comment]
Hi Jaymassena - can you advise what you did in this instance? Did you cater for this and allow this particular reported "violation"?

**Comment by Jaymassena** [comment]
Yes, we worked around the problem by ignoring the EMBEDDED_MOBILEPROVISION_SHOULD_BE_PRESENT violation.

**Comment by mrstrong79** [comment]
Ok - great, thanks for this!

**Comment by Juslwk** [comment]
@Jaymassena I have the exact same issue. Do you mind sharing where you added the code to ignore this violation?

**Comment by padresmurfa** [comment]
Hi all. (Obviously) I do not really maintain this project anymore. I'm out of Xamarin, C#, and have been for a few years, thus I do not even have the tools to build the project. I'm furthermore no longer in any way involved in any project that needs this library.

I'm glad that it has been of value to you however. At the moment, I'd recommend using Cryoprison as a library, rather than a Nuget package.
